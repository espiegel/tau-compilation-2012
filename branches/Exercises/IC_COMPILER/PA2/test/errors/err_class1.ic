
class Gelfand extends Ivanchuk {

 string foo(int a, string t) { return "aaa"; }
 void foo2(int a) { return; }
 void foo3() { return; }
 int foo4() { return 7; }
 
 static string foo5(int a) { return "aaa"; }
 static void foo6(int a) { return; }
 static void foo7(int[][] a) { return; }
 static int foo8() { return 7; }



