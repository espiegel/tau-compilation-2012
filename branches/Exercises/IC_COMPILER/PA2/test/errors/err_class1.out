16: Syntax error: at token EOF
